x/examples/mobileproxy/README.md

```diff
@@ -0,0 +1,175 @@
+# Local Proxy Library for Mobile Apps
+
+This package illustrates the use Go Mobile to generarate a mobile library to run a local proxy and configure your app networking libraries.
+
+### Build the Go Mobile binaries with [`go build`](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies)
+
+```bash
+go build -o ./out/ golang.org/x/mobile/cmd/gomobile golang.org/x/mobile/cmd/gobind
+```
+
+### Build the iOS and Android libraries with [`gomobile bind`](https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile#hdr-Build_a_library_for_Android_and_iOS)
+
+```bash
+PATH="$PATH:$(pwd)/out" $(pwd)/out/gomobile bind -target=ios -o "$(pwd)/out/LocalProxy.xcframework" github.com/Jigsaw-Code/outline-sdk/x/appproxy
```

Comment:
Since we already set the `PATH`, this is redundant?

```suggestion
PATH="$PATH:$(pwd)/out" gomobile bind -target=ios -o "$(pwd)/out/LocalProxy.xcframework" github.com/Jigsaw-Code/outline-sdk/x/appproxy
```

Reply:
Good point, fixed.
I switched the PATH order to make sure we use the gomobile in the PATH, not from the system-level install.